plugins/file-upload/src/helpers/upload.js

```diff
@@ -15,14 +18,30 @@ export const upload = async (config, file) => {
             const { baseURL, sasSignature, containerName } = config;
             const uploadUrl =  baseURL+containerName+'/'+file.name+sasSignature
             const downloadUrl = uploadUrl;
+
             return fetch(uploadUrl, {
                 method: 'PUT',
                 body: file,
                 headers: {
                     'x-ms-blob-type': 'BlockBlob'
                 },
             })
-                .then(() => downloadUrl )
+                .then(() => downloadUrl)
         }
+        case 'live-agent': {
+            const { userApiKey, host, accountId, conversationId } = config;
+            const uploadUrl = `${host}/api/v1/accounts/${accountId}/conversations/${conversationId}/messages`;
+            const form = new FormData();
+            form.append('attachments[]', file);
+            form.append('content', 'file-upload');
+
+            return Axios.post(uploadUrl, form, {
```

Comment:
Is it necessary to use Axios instead of fetch?

Reply:
I tried doing it with fetch for hours and unfortunately it didn't work, Axios is the only way it works even if all parameters are the same